webapp/src/sagas.js

```diff
@@ -339,4 +341,34 @@ function* handleIntentUnconfirmedBid(action) {
   }
 }
 
+const BID_INCREASE = 1.1
+
+function* handleFastBid(action) {
+  const parcel = action.parcel
+  if (parcel.projectId) {
+    // TODO: Show message, can't bid on project
+    return
+  }
+  const { x, y } = parcel
+  const amount = parcel.amount ? Math.ceil(parcel.amount * BID_INCREASE) : 1000
```

Comment:
`BID_INCREASE` already exists on `BidParcelModal` with another name, this can create confusion and be difficult to change in the future. How about we extract it?

On also, while it's probably not going to change, we use `ONE_LAND_IN_MANA` on the rest of the codebase to avoid having to juggle `1000` around.